backend/topping.ts

```diff
@@ -0,0 +1,12 @@
+import { ObjectId } from 'mongodb'
+
+export type Topping = {
+  _id: ObjectId,
+  name: string,
+  category: string,
+  img: string
+}
+
+export type ToppingWithCategoryId = Omit<Topping,'category'> & {
```

Comment:
what is Omit?

Reply:
`Omit<Topping,'category'>` is the same as `Topping` but without the `category` property

then the `& { category: number }` adds `category` back in, but as a number instead of a string